sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_utils.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import os
+
+from azure.monitor.opentelemetry.exporter._constants import (
+    _APPLICATIONINSIGHTS_STATS_CONNECTION_STRING_ENV_NAME,
+    _APPLICATIONINSIGHTS_STATS_LONG_EXPORT_INTERVAL_ENV_NAME,
+    _APPLICATIONINSIGHTS_STATS_SHORT_EXPORT_INTERVAL_ENV_NAME,
+    _DEFAULT_NON_EU_STATS_CONNECTION_STRING,
+    _DEFAULT_EU_STATS_CONNECTION_STRING,
+    _DEFAULT_STATS_SHORT_EXPORT_INTERVAL,
+    _DEFAULT_STATS_LONG_EXPORT_INTERVAL,
+    _EU_ENDPOINTS,
+    _REQ_DURATION_NAME,
+    _REQ_SUCCESS_NAME,
+)
+from azure.monitor.opentelemetry.exporter.statsbeat._state import (
+    _REQUESTS_MAP_LOCK,
+    _REQUESTS_MAP,
+)
+
+
+def _get_stats_connection_string(endpoint: str) -> str:
+    cs_env = os.environ.get(_APPLICATIONINSIGHTS_STATS_CONNECTION_STRING_ENV_NAME)
+    if cs_env:
+        return cs_env
+    for endpoint_location in _EU_ENDPOINTS:
+        if endpoint_location in endpoint:
+            # Use statsbeat EU endpoint if user is in EU region
+            return _DEFAULT_EU_STATS_CONNECTION_STRING
+    return _DEFAULT_NON_EU_STATS_CONNECTION_STRING
+
+
+# seconds
+def _get_stats_short_export_interval() -> int:
+    ei_env = os.environ.get(_APPLICATIONINSIGHTS_STATS_SHORT_EXPORT_INTERVAL_ENV_NAME)
+    if ei_env:
+        try:
+            return int(ei_env)
+        except ValueError:
+            return _DEFAULT_STATS_SHORT_EXPORT_INTERVAL
+    return _DEFAULT_STATS_SHORT_EXPORT_INTERVAL
+
+
+# seconds
+def _get_stats_long_export_interval() -> int:
+    ei_env = os.environ.get(_APPLICATIONINSIGHTS_STATS_LONG_EXPORT_INTERVAL_ENV_NAME)
+    if ei_env:
+        try:
+            return int(ei_env)
+        except ValueError:
+            return _DEFAULT_STATS_LONG_EXPORT_INTERVAL
+    return _DEFAULT_STATS_LONG_EXPORT_INTERVAL
+
+
+def _update_requests_map(type_name, value):
```

Comment:
Is there a specific reason for making value required?

Reply:
So we can consolidate the logic for when value is a number or a non-number.